ecomm-proj/src/components/Header.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+
+class Header extends React.PureComponent {
+  render () {
+    return (
+      <div>
+        <ul className='menu-list'>
```

Comment:
Out of scope but this should be yet another component, `<Nav />` or `<NavBar />`